conf/config-default.yaml

```diff
@@ -375,6 +375,7 @@ plugins:                          # plugin list (sorted by priority)
   - rocketmq-logger                # priority: 402
   - syslog                         # priority: 401
   - udp-logger                     # priority: 400
+  - file-logger                    # priority: 399
```

Comment:
The same type of plugin, the first number should be the same. `4xx` is for logger plugins

Reply:
> The same type of plugin, the first number should be the same. `4xx` is for logger plugins

Ignore this, it looks like it's not the rule anymore